### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by tomicapretto**
This PR contains a couple of small updates.

* Fix GitHub handles on a couple of notebooks.
* Add beta regression notebook to the webpage.
* Fix the model in the hierarchical binomial regression model. The previous model was badly specified.

### CODE REVIEW

File: bambi/backend/terms.py [hunk 1]
```diff
@@ -35,18 +35,25 @@ def build(self, spec):
         response_dims = []
         if spec.response.categorical and not spec.response.binary:
             response_dims = list(spec.response.pymc_coords)
+            response_dims_n = len(spec.response.pymc_coords[response_dims[0]])
+
+            # Arguments may be of shape (a,) but we need them to be of shape (a, b)
```
Comment (canyon289) [inline bambi/backend/terms.py:40]: Nice docstring

File: bambi/tests/test_built_models.py [hunk 2]
```diff
@@ -71,6 +71,15 @@ def inhaler():
     return data
 
 
+@pytest.fixture(scope="module")
+def categorical_family_categorical_predictor():
+    from os.path import dirname, join
```
Comment (canyon289) [inline bambi/tests/test_built_models.py:76]: Why import these here and not in global namespace?

**Review by canyon289 — APPROVED** [review]
i skimmed this but overall it looks good

File: bambi/tests/test_built_models.py (hunk 2 above)
Comment (tomicapretto) [reply bambi/tests/test_built_models.py:76]: There's no real reason. They just have been there forever. But this is a good time to move them to the global namespace.

**Review by aloctavodia — APPROVED** [review]

**Pull request merged**